scripts/build-app.sh

```diff
@@ -38,6 +38,5 @@ docker run --rm \
     -v $docker_auth_path:/root/.docker/config.json \
     -v $PWD/demo:/api/src/app/demo \
     -v $PWD/git:/api/src/app/git \
-    -v $PWD/demo:/api/src/customConfigTests/demo \
```

Comment:
The custom config tests don't actually get run in `orderly-web-app-build` so there was no need to mount the volume here. They actually get run here: https://github.com/vimc/orderly-web/blob/master/scripts/run-custom-config-tests-in-container.sh#L38